.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Bug report
+about: Create a report to report an error or unexpected behavior
```

Comment:
"Create a report to address/detail/discuss/introduce"

Reply:
Hi @vncoelho, I didn't understand. What do you mean? What would be the final phrase?
This is for bug report only, that is why I use the "report" word

Reply:
Just to remove the double use of the word "report". I think it will look better.

Reply:
Good eye @vncoelho , fixed